app/src/atoms/InlineNotification/index.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react'
+import {
+  Icon,
+  JUSTIFY_SPACE_BETWEEN,
+  IconProps,
+  Flex,
+  DIRECTION_ROW,
+  ALIGN_CENTER,
+  COLORS,
+  SPACING,
+  TYPOGRAPHY,
+  BORDERS,
+  Btn,
+} from '@opentrons/components'
+import { StyledText } from '../text'
+
+import type { StyleProps } from '@opentrons/components'
+
+type InlineNotificationType = 'alert' | 'error' | 'neutral' | 'success'
+
+export interface InlineNotificationProps extends StyleProps {
+  /** name constant of the icon to display */
+  type: InlineNotificationType
+  /** InlineNotification contents */
+  heading: string
+  message?: string
+  /** Optional dynamic width based on contents */
+  hug?: boolean
+  /** optional handler to show close button/clear alert  */
+  onCloseClick?: (() => void) | React.MouseEventHandler<HTMLButtonElement>
+}
+
+const INLINE_NOTIFICATION_PROPS_BY_TYPE: Record<
+  InlineNotificationType,
+  { icon: IconProps; backgroundColor: string; color: string }
+> = {
+  alert: {
+    icon: { name: 'ot-alert' },
+    backgroundColor: COLORS.yellow_three,
+    color: COLORS.yellow_two,
+  },
+  error: {
+    icon: { name: 'ot-alert' },
+    backgroundColor: COLORS.red_three,
+    color: COLORS.red_two,
+  },
+  neutral: {
+    icon: { name: 'information' },
+    backgroundColor: COLORS.darkBlack_twenty,
+    color: COLORS.darkBlackEnabled,
+  },
+  success: {
+    icon: { name: 'ot-check' },
+    backgroundColor: COLORS.green_three,
+    color: COLORS.green_two,
+  },
+}
+
+export function InlineNotification(
+  props: InlineNotificationProps
+): JSX.Element {
+  const { heading, hug = false, onCloseClick, message, type } = props
+  const fullHeading = `${heading}${message ? '. ' : ''}`
+  const fullmessage = `${message}.`
```

Comment:
```suggestion
  const fullmessage = `${message}`
```
or remove . from the passing props.

![Screenshot 2023-05-03 at 3 14 38 PM](https://user-images.githubusercontent.com/474225/236020870-0c36f8c1-eb34-458b-818a-0b85c5677fcc.png)

I also can see the extra . in videos you uploaded.

Reply:
D'oh! I need to get these new glasses adjusted.

Reply:
This turned out to be a stray period in the translation file. Fixed!

Reply:
cool, let me check quickly.

Reply:
The issue has been fixed.